README.md

```
@@ -20,6 +20,7 @@
   - [🛠 使用技術](#-使用技術)
   - [📡 インフラ構成図](#-インフラ構成図)
```

markdownlint-cli2 (0.20.0)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)